Mklinks subprocess -- rcsdiff -c -rR5-Vanilla -rR5-Mvex-r1 server/ddx/sun/RCS/Imakefile,v 
*** 1.4	1991/09/24 04:45:46
--- 1.5	1991/09/24 04:48:49
***************
*** 14,20 ****
  #endif
  
  SRCS1 =	sunInit.c sunBW2.c sunCursor.c sunIo.c sunKbd.c sunMouse.c sunUtils.c\
! 	sunCG.c sunCG2C.c sunCG2M.c sunCG3C.c sunCG4C.c sunCG6C.c $(GXSRC)\
   	sunKeyMap.c
  
  SRCS =	$(SRCS1) kbd_mode.c constype.c 
--- 14,20 ----
  #endif
  
  SRCS1 =	sunInit.c sunBW2.c sunCursor.c sunIo.c sunKbd.c sunMouse.c sunUtils.c\
! 	sunCG.c sunCG8C.c $(GXSRC)\
   	sunKeyMap.c
  
  SRCS =	$(SRCS1) kbd_mode.c constype.c 
***************
*** 22,28 ****
  OFILES = sunInit.o sunInitMono.o sunInitExtMono.o
  
  OBJS = 	sunBW2.o sunCursor.o sunIo.o sunKbd.o sunMouse.o sunUtils.o\
! 	sunCG.o sunCG2C.o sunCG2M.o sunCG3C.o sunCG4C.o sunCG6C.o $(GXOBJ)\
  	sunKeyMap.o
  
     INCLUDES = -I. -I../mfb -I../mi -I../cfb -I../../include\
--- 22,28 ----
  OFILES = sunInit.o sunInitMono.o sunInitExtMono.o
  
  OBJS = 	sunBW2.o sunCursor.o sunIo.o sunKbd.o sunMouse.o sunUtils.o\
! 	sunCG.o sunCG8C.o $(GXOBJ)\
  	sunKeyMap.o
  
     INCLUDES = -I. -I../mfb -I../mi -I../cfb -I../../include\
